crates/api/src/entrypoint.rs

```diff
@@ -1,30 +1,157 @@
 use crate::{deploy::DeployFile, manifest::WasmImage, wasmtime_unit::Wasmtime};
 
+use std::env;
 use std::path::{Component, Path, PathBuf};
 
 use anyhow::{bail, Result};
 use log::info;
 
-/// Validates the deployed image.
-///
-/// Takes path to the workdir as an argument.
-pub fn start(workdir: impl AsRef<Path>) -> Result<()> {
-    let workdir = workdir.as_ref();
-    let deploy_file = DeployFile::load(workdir)?;
+const INIT_MEM_VAR: &str = "YA_RUNTIME_WASI_INIT_MEM";
+const OPTIMIZE_VAR: &str = "YA_RUNTIME_WASI_OPT";
+const SGX_VAR: &str = "YA_RUNTIME_WASI_SGX";
 
-    info!(
-        "Validating deployed image {:?}.",
-        get_log_path(workdir, deploy_file.image_path())
-    );
+/// [Yagna] WASI runtime configuration.
```

Comment:
Is this a missing link in the docs? Why `[Yagna]`?

Reply:
fixed.